src/Features/Core/Portable/xlf/FeaturesResources.cs.xlf

```diff
@@ -562,6 +562,11 @@
         <target state="translated">Stream musí podporovat operace read a seek.</target>
         <note />
       </trans-unit>
+      <trans-unit id="Struct_contains_assigment_to_this_outside_the_scope_constructor_Make_readonly_fields_writable">
+        <source>Struct contains assignment to 'this' outside the scope of constructor. Make readonly fields writable</source>
+        <target state="translated">Struktura obsahuje přiřazení k „tomuto“ mimo rozsah konstruktoru. Umožněte zapisování polí jen pro čtení</target>
```

Comment:
did you manually translate these yourself? :)

you don't need to do that.  the loc team can do this here :)

Reply:
ok 😀

Reply:
can you undo these :)  these should jus tbe generated as english.  loc will take over.